src/main/kotlin/com/openlattice/organizations/mapstores/OrganizationExternalDatabaseColumnMapstore.kt

```diff
@@ -0,0 +1,79 @@
+package com.openlattice.organizations.mapstores
+
+import com.hazelcast.config.InMemoryFormat
+import com.hazelcast.config.MapConfig
+import com.hazelcast.config.MapIndexConfig
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.mapstores.TestDataFactory
+import com.openlattice.organization.OrganizationExternalDatabaseColumn
+import com.openlattice.postgres.PostgresTable
+import com.openlattice.postgres.ResultSetAdapters
+import com.openlattice.postgres.mapstores.AbstractBasePostgresMapstore
+import com.zaxxer.hikari.HikariDataSource
+import java.sql.PreparedStatement
+import java.sql.ResultSet
+import java.util.*
+
+const val ID_INDEX = "__key"
```

Comment:
Just a minor thing, but you can use `QueryConstants.KEY_ATTRIBUTE_NAME.value` here. That enum has the 2 constant index names for hazelcast maps.

Reply:
I tried to do this and kotlin was unhappy. Seems to think that `QueryConstants.KEY_ATTRIBUTE_NAME.value()` is not a constant.

Reply:
You don't need to put it into a constant anymore I think. But otherwise, you can just use the QueryConstants anywhere instead of creating a new field that references it

Reply:
replaced other constant with QueryConstant!